meerk40t/gui/wxmmain.py

```diff
@@ -325,6 +340,7 @@ def load_dialog(**kwargs):
                 if fileDialog.ShowModal() == wx.ID_CANCEL:
                     return  # the user changed their mind
                 pathname = fileDialog.GetPath()
+                gui.clear_and_open(pathname)
```

Comment:
This line doesn't refer to any current code. And a such would crash.

Reply:
I think it was in @jaredly 's previous PR.

Reply:
As do I, but it was sort of a fish out of water change there. And testing it crashed things. So it shouldn't have been included in the PR. There was another small bug where he loaded the `command` as an `int` when he used a string there that didn't really work. But, as far as PRs go this was/is decidedly quite robust. Most of my edge smoothing was commenting out parts of this. And adding in the routine to let you double-click to edit the values.